### ISSUE TITLE
[BUG] Black scanner page when bottom sheet enable

### ISSUE THREAD

**Issue by alavalmahesh** [juliuscanute/qr_code_scanner#292]
When I enable to bottom sheet on top of scanner page on android, scanner page is displaying the black screen, cannot able to reset the scanner after that.

**Comment by Parthib** [comment]
Yeah I'm getting the same issue. The camera opens once and then black screen from then onwards. It still seems to scan qr codes successfully despite the black view. In the logs I keep seeing ```W/System  (26368): A resource failed to call release.```

**Comment by jaleleddineagrebi** [comment]
we have the some issue can you take it on priority for next release version

**Comment by fede1295** [comment]
same issue for me... please fix it!

**Comment by nilevars** [comment]
We are facing the same issue now

**Comment by krishna700** [comment]
Adding `resizeToAvoidBottomInset: false,` in the Scaffold of the widget which contains the QRview will solve the black screen
